### PR TITLE
NextRoll Bidder Adapter

### PR DESCRIPTION
**Pull request by abijr**
## Type of change

- [x] New bidder adapter   

## Description of change

- test parameters for validating bids
```
{
  bidder: nextroll',
  params: {
      publisherId: "test_id",
      sellerId: "test_seller_id"
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- Maintainer: nextroll@prebid.org
- [x] official adapter submission

## Other information

Documentation PR: https://github.com/prebid/prebid.github.io/pull/1796

### CODE REVIEW

File: test/spec/modules/nextrollBidAdapter_spec.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+import * as utils from 'src/utils';
+
+const PUBTAG_LOCAL_STORAGE_KEY = 'nextroll_fast_bid';
+const PUBTAG_DATE_SUFFIX = '_set_date'
+
+describe('nextrollBidAdapter', function() {
+  let utilsMock;
+  beforeEach(function () {
+    // Remove to avoid side effects
+    localStorage.removeItem(PUBTAG_LOCAL_STORAGE_KEY);
```
Comment (jaiminpanchal27) [inline test/spec/modules/nextrollBidAdapter_spec.js]: Where are you trying to set these ? I did not find any localstorage usage anywhere

File: modules/nextrollBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+    { s: 'Windows Server 2003', r: /Windows NT 5.2/ },
+    { s: 'Windows XP', r: /(Windows NT 5.1|Windows XP)/ },
+    { s: 'UNIX', r: /UNIX/ },
+    { s: 'Search Bot', r: /(nuhk|Googlebot|Yammybot|Openbot|Slurp|MSNBot|Ask Jeeves\/Teoma|ia_archiver)/ }
+  ];
+  let cs = find(clientStrings, cs => cs.r.test(userAgent));
+  return cs ? cs.s : 'unknown';
+}
+
+function _parseUrl(url) {
```
Comment (jaiminpanchal27) [inline modules/nextrollBidAdapter.js]: Can you use functions from https://github.com/prebid/Prebid.js/blob/master/src/url.js

File: modules/nextrollBidAdapter.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+  },
+
+  /**
+   * Register the user sync pixels which should be dropped after the auction.
+   *
+   * @param {SyncOptions} syncOptions Which user syncs are allowed?
+   * @param {ServerResponse[]} serverResponses List of server's responses.
+   * @return {UserSync[]} The user syncs which should be dropped.
+   */
+  getUserSyncs: function (syncOptions, serverResponses, gdprConsent) {
```
Comment (jaiminpanchal27) [inline modules/nextrollBidAdapter.js]: You can remove this function if you don't need to do user sync

File: modules/nextrollBidAdapter.js [hunk 4]
```diff
@@ -0,0 +1,234 @@
+import * as utils from '../src/utils';
```
Comment (jaiminpanchal27) [inline modules/nextrollBidAdapter.js]: Can you update to add 'js' extension to all imports. We recently did a change here https://github.com/prebid/Prebid.js/pull/4876

**Comment by abijr** [comment]
Thank you @jaiminpanchal27! 

We've addressed the change request. Please let us know if there are any issues 😃

**Pull request merged**